public/js/test/mochitest/head.js

```diff
@@ -58,6 +87,17 @@ function _afterDispatchDone(store, type) {
   });
 }
 
+/**
+ * Wait for a specific action type to be dispatch.
+ * If an async action, will wait for it to be done.
+ *
+ * @memberof mochitest/waits
+ * @param {Object} dbg
+ * @param {String} type
+ * @param {Number} eventRepeat
+ * @return {Promise}
+ * @static
+ */
```

Comment:
I was thinking of doing it but I had this question.
@jasonLaster If I do convert the `head.js` type annotation to Flow, will I have to make other changes to make sure that the mochitest still run? Or can it handle it?